GitHeart/ImageService/ImageService.swift

```diff
@@ -20,6 +20,7 @@ class ImageService {
 
 extension ImageService: ImageProvider {
     func imageBy(url: URL, completion: @escaping (UIImage?) -> Void) -> ImageProviderTask? {
+      // Creating an UIImage object from data consumes some time, if I'm right.
```

Comment:
I haven't measured but on my iPhone 11 Pro the scrolling is smooth. I guess I can put the whole UIImage in the cache but it will significantly increase memory size required to keep enough images in the cache.